compiler/rustc_target/src/spec/windows_gnullvm_base.rs

```diff
@@ -1,4 +1,5 @@
-use crate::spec::{cvs, Cc, LinkerFlavor, Lld, TargetOptions};
+use crate::spec::{cvs, DebuginfoKind, LinkerFlavor, SplitDebuginfo, TargetOptions};
```

Comment:
```suggestion
use crate::spec::{cvs, CC, DebuginfoKind, LinkerFlavor, Lld, SplitDebuginfo, TargetOptions};
```

Oops, manual merge issue